hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/BuildInfoCollector.java

```diff
@@ -34,6 +36,15 @@
 
     static final int CLASSPATH_MAX_LENGTH = 100_000;
 
+    static String formatClassPath(String classpath) {
+        String[] classPathEntries = classpath.split(System.getProperty("path.separator"));
+        String shortenedEntries = Arrays.stream(classPathEntries)
+                .filter(cpEntry -> cpEntry.endsWith(".jar"))
+                .map(cpEntry -> cpEntry.substring(cpEntry.lastIndexOf(System.getProperty("file.separator")) + 1))
+                .collect(Collectors.joining(":"));
```

Comment:
Just curious.
Why do we join using `:` instead of `,` for example?

Reply:
That was my observation as well, checked the Client Versions impl. and `,` was used there as well.

Reply:
I changed it to ',' . Originally I thought the `:` would preserver backward-compatibility with any currently used scripts / reports.